kibana-reports/public/components/report_definitions/report_settings/report_settings.tsx

```diff
@@ -218,6 +237,22 @@ export function ReportSettings(props: ReportSettingProps) {
     }
   };
 
+  const handleNotebooksSelect = (e) => {
+    setNotebooksSourceSelect(e);
+    let fromInContext = false;
+    if (window.location.href.includes('?')) {
+      fromInContext = true;
+    }
+    if (e.length > 0) {
+      reportDefinitionRequest.report_params.core_params.base_url =
+        getNotebooksBaseUrlCreate(edit, editDefinitionId, fromInContext) + 
+        e[0].value;
```

Comment:
add parameter to redirect notebooks to output only?

Reply:
That change should be pending until we add the url parameter change to Notebooks, no?

Reply:
i was thinking if you do something like `/id?view=output` now, you don't have to send PR to reporting again after notebook changes. But you're right it's going to be a useless parameter now, so up to you

Reply:
It should end up being a small change so I don't think it'll be an issue to add that change to Reporting in a future PR